[HttpKernel] Explain how to define default value in MapQueryString (#19627)
symfony/symfony-docs: controller.rst

The default status code returned if the validation fails is 404.

If the request query string is empty and if you still need to have a valid DTO, you can

> define a default value in your controller method::

Reviewer: ```suggestion
define a default value to your controller parameter::
```

Reviewer 2: I tweaked this while merging ... but apparently, the correct preposition here is `for` instead of `to` 😊  Thanks!

    use App\Model\UserDto;
    use Symfony\Component\HttpFoundation\Response;
    use Symfony\Component\HttpKernel\Attribute\MapQueryString;

    // ...

    public function dashboard(
        #[MapQueryString] UserDTO $userDto = new UserDTO()
    ): Response
    {
        // ...
    }

In this case your DTO should have default values.

> 

Reviewer: I think this line may be superfluous 🙂

Reviewer 2: Yes, I removed it while merging. Thanks.
